src/components/home/home.tsx

```diff
@@ -40,7 +40,10 @@ export const Home: FC = () => {
           <div>State</div>
           <div>Zip</div>
         </div>
-        {contacts.map((contact, i) => (
+        {loading && contacts == undefined? 

```

Comment:
You don't have to worry about fixing it for this PR but you should always use === to make sure it checks for types